### PR TITLE
Fixes #6553: Allow rudder server to remotely run the agent

### PR DESCRIPTION
**Pull request by peckpeck**
http://www.rudder-project.org/redmine/issues/6553

Replacing previous PR: https://github.com/Normation/rudder-techniques/pull/662

### CODE REVIEW

File: initial-promises/node-server/common/1.0/cf-served.cf [hunk 1]
```diff
@@ -58,6 +58,11 @@ bundle server access_rules
         maproot => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}"))  },
         admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}"))  };
 
+    any::
+      # Allow server to remotely run the agent
+      "/var/rudder/cfengine-community/bin/cf-agent"
+        admit   => { host2ip("${server_info.cfserved}"), string_downcase(escape("${server_info.cfserved}")) };
```
Comment (amousset) [inline initial-promises/node-server/common/1.0/cf-served.cf]: Why using both the hostname and the ip and not only the ip?

Comment (peckpeck) [inline initial-promises/node-server/common/1.0/cf-served.cf]: Because cf-serverd asks its hostname to the client before accepting the connection.

**Comment by VinceMacBuche** [comment]
This is ok to me ! Thank @peckpeck ! Merging

**Comment by peckpeck** [comment]
PR replaced by https://github.com/Normation/rudder-techniques/pull/664

**Pull request closed without merging**